xbmc/cores/dvdplayer/DVDDemuxers/DVDDemuxVobsub.h

```diff
@@ -35,7 +35,7 @@ class CDVDDemuxVobsub : public CDVDDemux
   CDVDDemuxVobsub();
   virtual ~CDVDDemuxVobsub();
 
-  virtual bool          Open(const std::string& filename, const std::string& subfilename = "");
+  virtual bool          Open(const std::string& filename, int source, const std::string& subfilename = "");
```

Comment:
why not adding this as an additional Open? In this new method set source and call original Open